ndb/src/google/cloud/ndb/_eventloop.py

```diff
@@ -345,25 +345,70 @@ def get_event_loop():
     raise exceptions.AsyncContextError()
 
 
-def add_idle(*args, **kwargs):
-    raise NotImplementedError
+def add_idle(callback, *args, **kwargs):
+    """Helper function.
 
+    Calls :method:`EventLoop.add_idle` on current event loop.
 
-def queue_call(*args, **kwargs):
-    raise NotImplementedError
+    Raises:
+        exceptions.AsyncContextError: If called outside of a context
+            established by :func:`~google.cloud.ndb.async_context`.
+    """
+    loop = get_event_loop()
+    loop.add_idle(callback, *args, **kwargs)
+
+
+def queue_call(delay, callback, *args, **kwargs):
+    """Helper function.
+
+    Calls :method:`EventLoop.queue_call` on current event loop.
+
+    Raises:
+        exceptions.AsyncContextError: If called outside of a context
+            established by :func:`~google.cloud.ndb.async_context`.
+    """
+    loop = get_event_loop()
+    loop.queue_call(delay, callback, *args, **kwargs)
 
 
 def queue_rpc(*args, **kwargs):
     raise NotImplementedError
 
 
-def run(*args, **kwargs):
-    raise NotImplementedError
+def run():
+    """Helper function.
```

Comment:
It seems that there could be a better one-liner than "Helper function" (and that "Helper function" would apply to most functions / methods)

What about

```suggestion
    """Calls ``run()`` on the current event loop
```

as you mention below

Reply:
Ok.